src/textual_inputs/text_input.py

```diff
@@ -77,15 +71,24 @@ def __init__(
         placeholder: str = "",
         title: str = "",
         password: bool = False,
+        style: InputFieldStyle,
```

Comment:
This should be optional, with default styling for those who don't want to take advantage of this feature or are just starting to use the library.

Reply:
This is now optional 👍